src/testsuite/vec_f128_dummy.c

```diff
@@ -29,11 +29,11 @@
 #define __STDC_WANT_IEC_60559_TYPES_EXT__ 1
 #define __STDC_WANT_IEC_60559_FUNCS_EXT__ 1
 #endif
-#ifndef __clang__
+#ifndef PVECLIB_DISABLE_F128MATH
 /* Disable <math.h> for __clang__ because of a bug involving <floatn.h>
    incombination with -mcpu=power9 -mfloat128. This means that ISO
-   mandated <math.h> functions __finitef128, __isinff128, etc are not
-   available.  */ 
+   mandated <math.h> functions signbit(), isfinite(), isnormal(),
+   isinf(), isnan(), etc are not available for __float128.  */
```

Comment:
delete this comment?

Reply:
I think I will keep this until the issues is resolved.

It is not just that the TG (type generic) support for \__float128 is missing. Just including <math.h> with -mfloat128 enable kills the compile.